pom.xml

```diff
@@ -1,20 +1,16 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
   <modelVersion>4.0.0</modelVersion>
-  
-  <properties>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <skipIntegrationTests>true</skipIntegrationTests>
-    <jenkins-test-harness.version>2.23</jenkins-test-harness.version>
-    <java.level>7</java.level>
-    <jenkins.version>2.7.4</jenkins.version>
-  </properties>
-  
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.15</version>
```

Comment:
Now it's a downgrade, since pom is at 2.15

Reply:
Right, I messed up the merge